Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -538,7 +538,7 @@ extension SwiftLanguageService {
         return
       }
       do {
-        let snapshot = try await documentManager.latestSnapshot(document)
+        let snapshot = try await self.latestSnapshot(for: document)
```

Comment:
I think you also need to change the call to `documentManager.latestSnapshot` in line 547.